curses-client.py

```diff
@@ -486,7 +508,18 @@ def main(stdscr):
             curses.curs_set(0)  # Hide cursor
             settings(stdscr, interface)
             curses.curs_set(1)  # Show cursor again
-            
+        
+        elif char == 47:
+            # Display packet log
+            if display_log is False:
+                display_log = True
+                packetlog_win.addstr(1, 1, "Packet Log")
+                packetlog_win.box()
+                packetlog_win.refresh()
```

Comment:
This will persist the log when opening and closing

```suggestion
            if display_log is False:
                display_log = True
                if not packet_buffer:
                    packetlog_win.addstr(1, 1, "Packet Log")
                    packetlog_win.box()
                    packetlog_win.refresh()
                else:
                    update_messages_window()
```